### PR TITLE
docs: FAQs update

### LINKED ISSUE #1539: [content] Review FAQs
Review past audits for good FAQ candidates. 

Assess whether existing content is still useful or needs to be moved or deleted.

### PR DESCRIPTION
**Pull request by jillianhowarth**
Closes #1539 

Refreshing the FAQs page to be up-to-date and answer the questions that are frequently asked.

### CODE REVIEW

File: src/pages/help/faq/index.mdx [hunk 1]
```diff
@@ -5,16 +5,25 @@ description:
...
 
-What is the Carbon Design System?
+<AnchorLink>About Carbon</AnchorLink>
+<AnchorLink>Contributing</AnchorLink>
+<AnchorLink>Creating with Carbon</AnchorLink>
 
-</PageDescription>
+</AnchorLinks>
+
+## All about Carbon
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:16]:
```suggestion
## About Carbon
```

File: src/pages/help/faq/index.mdx [hunk 2]
```diff
@@ -5,16 +5,25 @@ description:
...
+
+### What is the Carbon Design System?
 
 Carbon is the open-source design system for all IBM software products. It is a
 series of individual styles, components, and guidelines used for creating
 unified product experiences.
 
+To learn more about how we work check out
+[All about Carbon](/all-about-carbon/what-is-carbon).
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:25]:
```suggestion
[What is Carbon](/all-about-carbon/what-is-carbon).
```

File: src/pages/help/faq/index.mdx [hunk 3]
```diff
@@ -5,16 +5,25 @@ description:
...
+
 ### Who works on Carbon?
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:27]: Let's take out the sentence about the Austin studio. (I can't edit that paragraph for some reason.)

File: src/pages/help/faq/index.mdx [hunk 4]
```diff
@@ -25,36 +34,22 @@ who contribute back to
 and
 [carbon-react](https://github.com/carbon-design-system/carbon-components-react#contributors).
 
+## Contributing
+
 ### How can I contribute and/or propose new components/ideas?
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:39]:
```suggestion
### How can I contribute and/or propose new components/ideas?
```
```suggestion
### How can I contribute and/or propose new components or ideas?
```

File: src/pages/help/faq/index.mdx [hunk 5]
```diff
@@ -25,36 +34,22 @@ who contribute back to
...
-vanilla JavaScript library; that's written in ES6+, SCSS, and HTML and ships
-deliverables in vanilla JavaScript in ES5 as well as ES6, CSS, SCSS and HTML.
-Another is a React library that ships a React rewrite of the JavaScript, but
-uses the same CSS as the core library. There are also Angular and Vue libraries
-that use the same CSS.
-
 ### I see a bug. How do I report it?
 
 First, make sure the problem is reproducible. Once confirmed, open an issue in
 the appropriate GitHub repo (see below). We will address the bug as soon as we
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:47]:
```suggestion
the appropriate GitHub repo from the following list. We will address the bug as soon as we
```

File: src/pages/help/faq/index.mdx [hunk 6]
```diff
@@ -146,17 +138,11 @@ to explore the work so far.
...
-If keeping the number of columns unchanged is not important to you, see below
-examples.
-
-<Video vimeoId="356743483" />
-<Video vimeoId="356743492" />
-<Video vimeoId="356743508" />
-
-If keeping the number of columns to three is essential, here are some other
-options.
+### I haven't updated to the latest version of Carbon yet, and I still need the old documentation. Where can I find it?
```
Comment (janchild) [inline src/pages/help/faq/index.mdx:141]:
```suggestion
### Where can I find documentation for earlier versions of Carbon?
```

**Review by janchild — COMMENTED** [review]
Some little tweaks here!

File: src/pages/help/faq/index.mdx (hunk 3 above)
Comment (jillianhowarth) [reply src/pages/help/faq/index.mdx:27]: Done!

**Review by janchild — APPROVED** [review]
🥂 Thank you, @jillianhowarth! A great improvement.

**Pull request merged**